<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADCCalcPanel</class>
 <widget class="QWidget" name="ADCCalcPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>标定页面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>当前k</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dsb_current_k">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>当前b</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dsb_current_b">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>显示电压</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>实际电压</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>标定点1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dsb_display_v_1">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="dsb_real_v_1">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>标定点2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dsb_display_v_2">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="dsb_real_v_2">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_calc">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>新k</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dsb_new_k">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>新b</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dsb_new_b">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_apply">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
